src/commands/messageContent/AutoReplace.ts

```diff
@@ -0,0 +1,60 @@
+import { Message } from "discord.js";
+
+import ContentCommand from "./ContentCommand";
+
+type AutoReplaceCategory = {
+  name: string;
+  reason: string;
+  replacements: RegExp[];
+  combinedReplacement?: RegExp;
+}
+
+const autoReplaceStrings: AutoReplaceCategory[] = [
+  {
+    name: "twitter",
+    reason: "replacing links so we don't send traffic to the bad site:",
+    replacements: [
+      /twitter\.com/i,
+      /x\.com/i,
```

Comment:
netflix.com and other sites that end in x.com will get caught in this

Reply:
https://krebsonsecurity.com/2024/04/twitters-clumsy-pivot-to-x-com-is-a-gift-to-phishers/